docs/roadmap/volume-manager.rst

```diff
@@ -1,73 +0,0 @@
-Flocker Volume Manager
```

Comment:
No, the whole of the /roadmap folder has been removed. I've created FLOC 2010 with the contents of /roadmap attached, so this isn't lost when this branch is merged. Under this issue, it can be reviewed whether we want to retain this content, but the plan was to not include areas for potential development in the v1.0 docs.